django_tables2/rows.py

```diff
@@ -71,12 +71,20 @@ def table(self):
         '''
         return self._table
 
+    def get_even_odd_css_class(self):
+        '''
+        Return css class `even` or `odd`.
```

Comment:
maybe the word alternatingly or something like that should be added to this docstring?